src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Threading;
+using Xunit;
+
+namespace Test
+{
+    [Trait("category", "outerloop")]
+    public partial class ParallelQueryCombinationTests
+    {
+        [Theory]
+        [MemberData("UnaryOperators")]
+        [MemberData("BinaryOperators")]
+        public static void Aggregate_OperationCancelledException(LabelledOperation source, LabelledOperation operation)
+        {
+            CancellationTokenSource cs = new CancellationTokenSource();
```

Comment:
Although seeing your usage of it further in the tests, probably makes sense to stick with what you have.  You're verifying that the OCE's that emerge contain the same token, and that can be better verified when the CT is associated with a CTS instance as you have it.